scripts/zones/Kuftal_Tunnel/mobs/Deinonychus.lua

```diff
@@ -14,7 +14,7 @@ entity.onMobDeath = function(mob, player, optParams)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.YOWIE_PH, 5, math.random(7200, 28800)) -- 2 to 8 hours
+    xi.mob.phOnDespawn(mob, ID.mob.YOWIE_PH, 5, 7200) -- 2 hour minimum
```

Comment:
Could you link where this time is coming from? I don't even see evidence for the original 2 to 8 anywhere obvious

Reply:
Lottery spawns should not have a window timer, they should simply be open after x amount of time. This is in line with pretty much the rest of the lottery NMs and specifically the rest of the lotto NMs in this zone. I'm not sure where the original source for this came from, but its basically 2 hours cooldown and then its open. If it takes 8 hours to pop, that's the nature of lotto spawns.